src/com/ppm/integration/agilesdk/connector/agilecentral/RallyIntegrationConnector_ko.properties

```diff
@@ -1,54 +1,13 @@
-THEME_STATUS=\ud14c\ub9c8 \uc0c1\ud0dc
-FEATURE_STATUS=\uae30\ub2a5 \uc0c1\ud0dc
-NO_CHARTS_BY_TECH_LIMIT=\uc77c\ubd80 \uae30\uc220\uc801\uc778 \uc81c\ud55c\uc73c\ub85c \uc778\ud574 \ucc28\ud2b8\ub97c \uc791\uc5c5 \uc138\ubd80 \uc0ac\ud56d \ud398\uc774\uc9c0\uc5d0 \ud45c\uc2dc\ud560 \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-CLICK_TO_SHOW_CHARTS_IN_NEW_WINDOW=\ucc28\ud2b8\ub97c \uc0c8 \ucc3d\uc5d0 \ud45c\uc2dc\ud558\ub824\uba74 \uc5ec\uae30\ub97c \ud074\ub9ad\ud558\uc2ed\uc2dc\uc624.
-AGM_CHARTS=HPE Agile Manager \ucc28\ud2b8
-ERROR_UNKNOWN_HOST_ERROR=\uc54c \uc218 \uc5c6\ub294 \ud638\uc2a4\ud2b8 \"{0}\"
-ERROR_UNKNOWN_ERROR=\uc54c \uc218 \uc5c6\ub294 \uc624\ub958 \"{0}\"
-ERROR_CONNECTIVITY_ERROR=\uc5f0\uacb0 \uc624\ub958
-ERROR_DOMAIN_NOT_FOUND=\"{0}\" \ub3c4\uba54\uc778\uc744 \ucc3e\uc744 \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-ERROR_PROJECT_NOT_FOUND=\"{0}\" \ud504\ub85c\uc81d\ud2b8\ub97c \ucc3e\uc744 \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-ERROR_RELEASE_NOT_FOUND=\"{0}\" \ub9b4\ub9ac\uc2a4\ub97c \ucc3e\uc744 \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-ERROR_AUTHENTICATION_FAILED=\uc778\uc99d \uc2e4\ud328
-ERROR_RELEASE_IS_NOT_STARTED=\"{0}\" \ub9b4\ub9ac\uc2a4\uac00 \uc2dc\uc791\ub418\uc9c0 \uc54a\uc558\uc2b5\ub2c8\ub2e4.
-ERROR_RELEASE_IS_FINISHED=\"{0}\" \ub9b4\ub9ac\uc2a4\uac00 \uc644\ub8cc\ub418\uc5c8\uc2b5\ub2c8\ub2e4.
-ERROR_CANNOT_GET_CURRENT_SPRINT=\ud604\uc7ac \uc2a4\ud504\ub9b0\ud2b8\ub97c \uac00\uc838\uc62c \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-WARN_TM_AGM_RELEASE_NOT_ALIGNED=AGM\uc5d0\uc11c PPM \ud0c0\uc784\uc2dc\ud2b8 \uae30\uac04\uc5d0 \ub9de\ucdb0\uc9c0\uc9c0 \uc54a\uc740 \ub9b4\ub9ac\uc2a4\ub97c \ub098\ud0c0\ub0c5\ub2c8\ub2e4.
-ERROR_HTTP_CONNECTIVITY_ERROR=\uc5f0\uacb0 \uc624\ub958 \"{0}\"
-ERROR_TIMESHEET_RETRIEVE=\ud0c0\uc784\uc2dc\ud2b8 \ub370\uc774\ud130 \uac80\uc0c9 \uc624\ub958 \"{0}\"
-ERROR_ACCESS_TOKEN_RETRIEVE=\uc561\uc138\uc2a4 \ud1a0\ud070\uc744 \uac00\uc838\uc62c \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-CLIENT_ID_OR_SECRET_KEY_NOT_SET=\ud074\ub77c\uc774\uc5b8\ud2b8 ID \ub610\ub294 \uc554\ud638 \ud0a4\uac00 \uc124\uc815\ub418\uc9c0 \uc54a\uc558\uc2b5\ub2c8\ub2e4.
-#for create release fields
-ERROR_WORKSPACE_ID=\uc791\uc5c5 \uacf5\uac04 ID\ub97c \uac00\uc838\uc62c \uc218 \uc5c6\uc2b5\ub2c8\ub2e4.
-IS_CREATE_RELEASE=\uc0c8 \ub9b4\ub9ac\uc2a4\ub97c \uc791\uc131\ud558\uc2dc\uaca0\uc2b5\ub2c8\uae4c?
-CREATE_RELEASE_NAME=\uc774\ub984
-CREATE_RELEASE_DESCRIPTION=\uc124\uba85
-CREATE_RELEASE_START_TIME=\uc2dc\uc791 \uc2dc\uac04
-CREATE_RELEASE_END_TIME=\uc885\ub8cc \uc2dc\uac04
-CREATE_RELEASE_SPRINT_DURATION=\uc2a4\ud504\ub9b0\ud2b8 \uae30\uac04
-CREATE_RELEASE_SPRINT_DURATION_UNIT=\uc2a4\ud504\ub9b0\ud2b8 \uae30\uac04 \ub2e8\uc704
-CREATE_RELEASE_DAY=\uc77c
-CREATE_RELEASE_WEEK=\uc8fc
-NO_DATA_AVAILABLE=\uc0ac\uc6a9 \uac00\ub2a5\ud55c \ub370\uc774\ud130\uac00 \uc5c6\uc2b5\ub2c8\ub2e4.
+USERNAME=Username
+PASSWORD=Password
+PROXY_HOST=Proxy Host
+PROXY_PORT=Proxy Port
+SUBSCRIPTION=Subscription
+WORKSPACE=\uc791\uc5c5 \uacf5\uac04
```

Comment:
Do we also give up Korean? 

If we give up a translation, it would be better to remove the translation files altogether instead of filling them with the english translations.